src/cljc/proton/diff.cljc

```diff
@@ -0,0 +1,38 @@
+(ns proton.diff
+  (:require
+   [clojure.data :refer [diff]]
+   [clojure.pprint :refer [pprint]]
+   [clojure.string :as string]))
+
+(defn- blank-padding [origin size]
+  (into origin (take size (repeat ""))))
+
+(defn- align-length [a-diff b-diff]
+  (let [ac (count a-diff)
+        bc (count b-diff)]
+    (case (compare ac bc)
+      1 [a-diff (blank-padding b-diff (- ac bc))]
```

Comment:
Can you point to something that says this is always the case?
If you want to be conservative, consider using pos?/neg? instead of only considering 1/-1

```
(defn compare
  "Comparator. Returns a negative number, zero, or a positive number
  when x is logically 'less than', 'equal to', or 'greater than'
  y. Same as Java x.compareTo(y) except it also works for nil, and
  compares numbers and collections in a type-independent manner. x
  must implement Comparable"
  {
   :inline (fn [x y] `(. clojure.lang.Util compare ~x ~y))
   :added "1.0"}
  [x y] (. clojure.lang.Util (compare x y)))
```

Reply:
When `x` and `y` are integer, `compare` return only `1` , `-1`  and `0`.
However, I'll use `pos?` `neg?` because it may cause bugs later.